go/vt/tabletserver/sqlquery.go

```diff
@@ -673,3 +672,30 @@ func withTimeout(ctx context.Context, timeout time.Duration) (context.Context, c
 	}
 	return context.WithTimeout(ctx, timeout)
 }
+
+func getColumnType(qre *QueryExecutor, columnName, tableName string) (int64, error) {
+	conn, err := qre.getConn(qre.qe.connPool)
+	if err != nil {
+		return mproto.VT_NULL, err
+	}
+	defer conn.Recycle()
+	query := fmt.Sprintf("SELECT %v FROM %v LIMIT 0", columnName, tableName)
```

Comment:
Both has been validated in QuerySplitter (query_splitter.go) before using in getColumnType.

table name validation: https://github.com/youtube/vitess/blob/master/go/vt/tabletserver/query_splitter.go#L71
column name validation: https://github.com/youtube/vitess/blob/master/go/vt/tabletserver/query_splitter.go#L82